src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ArrivalEntity.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import java.time.LocalDate
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "arrival")
+data class ArrivalEntity(
```

Comment:
This is more of potential gotcha and not saying the affects you.

With data classes you get toString for free. You may (or may not) want to override this in case you have any PII on your entities that could be leaked to the logs by mistake.

Reply:
This is a really good point, have changed the toString on them all